app/mailers/user_mailer.rb

```diff
@@ -44,4 +44,28 @@ def api_token_granted_notification(user)
              subject: "#{_('API rights in')} #{Rails.configuration.branding[:application][:name]}")
       end
   end
+  
+  def feedback_notification(user, plan)
+    @user = user
+
+    if user.org.present?
+      @org = org
+      @plan = plan
+      
+      # Use the generic feedback message unless the Org has specified one
+      subject = "#{_('A user has requested feedback for their DMP in')} #{Rails.configuration.branding[:application][:name]}"
+      subject = org.feedback_email_subject if org.feedback_email_subject.present?
+      
+      # Send an email to all of the org admins as well as the Org's administrator email
+      emails = user.org.users.select{ |usr| usr.can_org_admin? && usr != user }

```

Comment:
why don't you use a scoped query to select can_org_admin users distinct from current. If the org has a considerable amount of users this would not scale...